event-streams/src/event-streams-v1.0-wd01.md

```diff
@@ -224,92 +225,304 @@ Rescorla, E. and B. Korver, "Guidelines for Writing RFC Text on Security
+  events across multiple partitions.
+
+# 3 Capabilities and Default Behaviors 
+
+## 3.1 Connection and Link Capabilities
+
+On connection establishment, the event log node MUST indicate its support of
+this specification through the exchange of a connection capability (see Section
+2.7.1, [AMQP](#amqp-v10)). The capability allows for the consumer or producer to detect
+whether extensions as defined in this specification are understood and may be
+used.
+
+| Capability Name         | Definition
+| ------------------------|---------------------------------------------------------------- |
+| AMQP_EVENT_STREAMS_V1_0 | If present in the offered-capabilities field of the open frame, the sender of the open supports the use of event streams extensions. |
+
+## 3.2 Default Behaviors
+
+This specification does not modify or override any rules of the core AMQP 1.0
+specification.
+
+Even with the capability being announced at the connection level, all additional
+behaviors defined here are optional, which means that a producer or consumer
+MUST be able to successfully interact with a conforming event log implementation
+just as with any regular AMQP node, for instance:
+
+* Producer and consumers that do not choose a partition on a partitioned event
+  log are either assigned a partition by the event log node, or their links are
+  left partition-agnostic.
+* Producers are not required to annotate events with partition hints, even on
+  partition-agnostic links. The event log node chooses the partition in absence
+  of such hints.
+* Consumers are not required to provide offset filter information. If no filter
+  is provided, the consumer is eligible to receive events accepted by the event
+  log node after the link was attached.
+* Consumers can rely on all existing AMQP mechanisms for delivery-state tracking
+  provided by the AMQP source terminus of the event log node. This means that
+  after attaching a link, the consumer is not required to keep track of offsets
+  for initiating further transfers while the link exists. The source might even
+  offer link durability [(AMQP 1.0, 3.5.5)](#amqp-v10) with full recovery of
+  unsettled state in the case of disconnects.
+
+# 4 Partition support
+
+An event log maintains a sequence of events. An event log MAY be split into
+multiple parallel partitions that are reachable under the same network address,
+or the responsibility for maintaining an event log might also be shared across
+multiple AMQP containers/nodes that each have a separate network address and are
+each responsible for one or more partitions of the event log.
+
+Producers and consumers attach to the event log via AMQP links. Those links MAY
+be bound to one, specific partition (partition-bound) or they MAY be
+partition-agnostic. On partition-agnostic links, producers MAY provider
+transfer-level hints for which partition the transfer ought to be routed to.
+
+This model allows for a range of negotiation options for how consumers are
```

Comment:
"... how producers and consumers are ..." not just "consumers" because in the following list you are describing options for both.